apps/accounts/serializers.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.auth import get_user_model
+
+from rest_framework import serializers
+
+# Get the UserModel
+UserModel = get_user_model()
+
+
+class UserDetailsSerializer(serializers.ModelSerializer):
+    """
+    Make username as a read_only field.
+    """
+    class Meta:
+        model = UserModel
```

Comment:
We could directly do `model=get_user_model()` instead of storing it in another variable.

Reply:
@deshraj Done!